.github/workflows/openscap.yml

```diff
@@ -35,13 +44,15 @@ jobs:
             exit "0"
           fi
 
+      # this is not ideal since it creates a PR per release
+      # with potentially 3 prs per workflow.
```

Comment:
This will be pain and rebasing hell, could you explore creating artifacts from the jobs and then a separate job that depends on all, downloads all artifacts and make one PR?

Reply:
Yeah agreed, that was the next step because it would be a massive pain. I have a meeting with the OpenSCAP team next week about this, they're split about using centos for generating RHEL customizations... so I'll put the changes on pause for now.